doc/mgmt/Docker to Host communication.md

```diff
@@ -2,7 +2,7 @@
 Docker to Host Communications
 
 # High Level Design Document
-#### Rev 0.7
+#### Rev 0.8
```

Comment:
Original document mentioned "translib API", is it really a dependency for applications?

Reply:
Management framework uses translib API to invoke dbus service, we can invoke dbus service directly.